lib/rs/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "rift"
+description = "Rust Thrift library"
+version = "0.5.1"
+license = "Apache-2.0"
+authors = ["Allen George <allen.george@gmail.com>"]
+readme = "README.md"
+documentation = "https://docs.rs/rift/"
+exclude = ["Makefile*"]
+keywords = ["thrift"]
+
```

Comment:
I have changed it to "Apache Thrift Developers <dev@thrift.apache.org>". This is what other author tags read, so I assume that's OK.